app/models/concerns/gobierto_common/sluggable.rb

```diff
@@ -35,5 +36,11 @@ def set_slug
 
       self.slug = new_slug
     end
+
+    def add_archived_to_slug
+      unless destroyed?
+        update_attribute(:slug, slug + "-slug-archived")
```

Comment:
I'd change the slug format to something like "archived-<superlong random string>", to avoid the rollbar I mentioned in the comments